src/tree_cmd.cpp

```diff
@@ -193,11 +224,19 @@ static void PlaceTreeGroups(uint num_groups)
 			int x = GB(r, 0, 5) - 16;
 			int y = GB(r, 8, 5) - 16;
 			uint dist = abs(x) + abs(y);
+			uint max_dist = MAX_DISTANCE_FROM_CENTER;
+
+			if (_settings_newgame.game_creation.tree_placer == TP_IMPROVED) {
```

Comment:
Needs a comment explaining why the distance is different when planting forests, I think.

Reply:
I'm not actually sure myself. shall i remove this part?